client/Dockerfile-client

```diff
@@ -0,0 +1,40 @@
+# Our First stage, that builds the application
+FROM node:12.13.0-alpine AS react-builder
+WORKDIR app
+
+# Copy all files 
+COPY . .
+
+# Install dependencies
+RUN yarn
```

Comment:
Because docker caches the layers (each docker statement creates a new layer) it would faster to build subsequent images as below:

```
# Install dependencies
COPY package.json yarn.lock /app
RUN yarn install

# Copy all files
COPY . .
```
Now `yarn install` will get executed only if the dependencies change.